src/js/jsx/Scrim.jsx

```diff
@@ -274,6 +319,40 @@ define(function (require, exports, module) {
             );
         }
     });
-
-    module.exports = Scrim;
+    
+    var canDropGraphic = function () {
```

Comment:
JSDoc for this please, also, let's keep this function definition near top of the file, prepend `_` and mark it private in JSDoc.